src/main/java/seedu/address/ui/EventTaskListCard.java

```diff
@@ -34,7 +35,7 @@ public EventTaskListCard(EventTask eventTask, int index) {
             nameLabel.setText(eventTask.getName().toString());
             startLabel.setText(eventTask.getStart().toString());
             endLabel.setText(eventTask.getEnd().toString());
-            if (eventTask.getEnd().isBefore(LocalDateTime.now())) {
+            if (!(new EventNotInPastPredicate()).test(eventTask)) {
```

Comment:
Here I notice that `EventNotInPastPredicate()` is not a good name at all. Given that we want to check if an event is "finished", a much better name would be `EventFinishedPredicate`.

That said, it would be even better if I could just do `eventTask.isFinished()` as well.